examples/create-file.js

```diff
@@ -9,8 +9,8 @@ const filename = path.join(__dirname, 'created-logfile.log');
 //
 // Remove the file, ignoring any errors
 //
-try { fs.unlinkSync(filename); }
-catch (ex) { }
+/* eslint-disable-next-line no-empty */
+try { fs.unlinkSync(filename); } catch (ex) {}
```

Comment:
It's not clear why this consolidation is helpful.
Where linter/compiler overrides are actually necessary, it's best to put less on each line, minimizing the amount subject to the exception; this commit seems to do the opposite.
This comment also applies below.